cloud-data-protection/src/components/login.tsx

```diff
@@ -0,0 +1,54 @@
+import React, {useEffect, useState} from "react";
+import './login.css';
+import {AuthService} from "../services/authService";
+import {Button, Divider, Input, InputLabel, Typography} from "@material-ui/core";
+import {useSnackbar} from 'notistack';
+import {http} from "../common/http";
+
+const Login = () => {
+    const [email, setEmail] = useState('');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
    const [email, setEmail] = useState("");
```